cmd/aries-js-worker/src/agent-rest-client.js

```diff
@@ -107,6 +107,15 @@ const pkgs = {
             method: "GET",
             pathParam:"id"
         },
+        GetCredentialByName: {
+            path: "/verifiable/credential/name/{name}",
```

Comment:
URL is `/verifiable/credential/name/{name}??` 

why can't be `/verifiable/credential/{name}?` like connection by ID in didexchange controller?

Reply:
we already have one API for ID - `/verifiable/credential/{id}`

Reply:
QueryConnectionByID: {
            path: "/connections/{id}",
            method: "GET",
            pathParam:"id"
        },
        QueryConnections: {
            path: "/connections",
            method: "GET"
        },

Reply:
https://github.com/hyperledger/aries-framework-go/blob/master/cmd/aries-js-worker/src/agent-rest-client.js#L105-L118.

Reply:
I can see ` path: "/verifiable/credential/{id}"` taken of ID?

May be we can follow `QueryConnectionByID vs QueryConnections` pattern from didexchange (may be a followup?)

Reply:
I don't think its applicable her. Here we have 3 APIs - GetByID, GetByName and GetAll.